### PR TITLE
Fix html encoding with brainwashing objectives in tgui

### PR DESCRIPTION
**Pull request by Absolucy**
## About The Pull Request

objectives are sanitized html and should be treated as such

## Why It's Good For The Game

readability bugfix

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
fix: Fix html encoding (which messed up apostrophes and such) with brainwashing objectives in the tgui popup
/:cl:

### CODE REVIEW

File: tgui/packages/tgui/interfaces/AntagInfoBrainwashed.tsx [hunk 1]
```diff
@@ -57,7 +58,13 @@ const ObjectivePrintout = (props, context) => {
           objectives.map((objective) => (
             <>
               <Stack.Item key={objective.count}>
-                {objective.count}. {objective.explanation}
+                {objective.count}.{' '}
+                <span
+                  // eslint-disable-next-line react/no-danger
+                  dangerouslySetInnerHTML={{
+                    __html: sanitizeText(objective.explanation), // brainwashing objectives are sanitized anyways
+                  }}
```
Comment (PowerfulBacon) [inline tgui/packages/tgui/interfaces/AntagInfoBrainwashed.tsx:66]: I dont like that players are allowed to use any HTML tags at all in their thing. This should be decoded on the byond side before being sent in as TGUI will sanitise it properly then

Comment (Absolucy) [reply tgui/packages/tgui/interfaces/AntagInfoBrainwashed.tsx:66]: It's sanitized on the BYOND side - this is due to the fact some objectives, like with cult, have `<b>` for example.

Comment (itsmeow) [reply tgui/packages/tgui/interfaces/AntagInfoBrainwashed.tsx:66]: Yea this is the correct way of doing this. Players can't write objective text most of the time and we already strip the input if so. sanitizeText adds extra safety by stripping scripts, links etc

**Review by itsmeow — APPROVED** [review]

**Comment by itsmeow** [comment]
For safety sake please test with tag injections

**Pull request merged**